cloudnetpy/plotting/plotting.py

```diff
@@ -140,6 +178,10 @@ def generate_figure(
         else:
             _plot_colormesh_data(ax, field, name, ax_value)
     case_date = set_labels(fig, axes[-1], nc_file, sub_title)
+    #    if addsources:
+    #        display_datasources(fig, axes[-1], nc_file)
```

Comment:
Correct, was intially planning to add this in the end because I only make 1var plots but then realised it should be for each axis so now this has been removed.